pulsar-client/src/main/java/com/yahoo/pulsar/client/api/ConsumerConfiguration.java

```diff
@@ -171,4 +173,24 @@ public ConsumerConfiguration setConsumerName(String consumerName) {
         this.consumerName = consumerName;
         return this;
     }
+    
+    public int getPriorityLevel() {
+        return priorityLevel;
+    }
+
+    /**
+     * Sets priority level for the shared subscription consumers to which broker gives more priority while dispatching
+     * messages.
+     * </p> 
+     * In Shared subscription mode, broker will first dispatch messages to upper priority-level consumers if they
```

Comment:
>I think it would be more intuitive to use descending priorities. (eg: 0=max-priority, 1, 2, ).

Actually, right now that's how the implementation is: 0=max-priority, 1=second, 2=..
[sorting](https://github.com/rdhabalia/pulsar/blob/854494db57e486109358090e20de7818e88c28ea/pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/persistent/PersistentDispatcherMultipleConsumers.java#L98): `consumerList.sort((c1, c2) -> c1.getPriorityLevel() - c2.getPriorityLevel());`

I will update the documentation part to make it more clear.